src/mpid/ch4/src/ch4_progress.h

```diff
@@ -101,44 +103,69 @@ MPL_STATIC_INLINE_PREFIX int MPIDI_Progress_vci_shm(int vci)
 }
 
 #undef FUNCNAME
-#define FUNCNAME MPIDI_Progress_test
+#define FUNCNAME MPIDI_Progress_vci
```

Comment:
Can you call it `MPIDI_Progress_test_vci` to make it clear that it's a single-pass routine?